homeassistant/components/sensor/influxdb.py

```diff
@@ -105,6 +105,15 @@ def __init__(self, hass, influx_conf, query):
                     query.get(CONF_FIELD),
                     query.get(CONF_MEASUREMENT_NAME),
                     query.get(CONF_WHERE))
+
+        _LOGGER.info("Rendering query: %s", formated_query)
+        query_template = template.Template(formated_query, hass)
```

Comment:
You should only instantiate templates using voluptuous. So use `cv.template` as a validator and set the hass object on it if it is not None.